### ISSUE TITLE
-nfpva does not work in certain cases with multiple parameters

### ISSUE THREAD

**Issue by ecos-platypus** [perltidy/perltidy#73]
The configuration option -nfpva has been added for issue #53
While it works in most cases, it seems to be confused by  some cases with multiple parameters.

### Working

```
$self -> method ('parameter_0') ;
$self -> method_with_long_name ('parameter_0') ;

$self -> method ( 'parameter_0', 'parameter_1' ) ;
$self -> method_with_long_name ('parameter_0') ;

$self -> method ('parameter_0') ;
$self -> method_with_long_name ( 'parameter_0', 'parameter_1' ) ;
```

### Not working

Expected behavior:
```
$self -> method ( 'parameter_0', 'parameter_1' ) ;
$self -> method_with_long_name ( 'parameter_0', 'parameter_1' ) ;
```

Actual behavior:
```
$self -> method                ( 'parameter_0', 'parameter_1' ) ;
$self -> method_with_long_name ( 'parameter_0', 'parameter_1' ) ;
```

**Comment by shancock9** [comment]
This will be fixed in the next release, which is about 2 weeks off.  It is fixed in the development version 20210717.03 which can be obtained at 

https://github.com/perltidy/perltidy/releases

**Comment by ecos-platypus** [comment]
Thanks for the quick fix, works like a charm!
